tests/snapshots/test_app/test_ask_prompt_template/client0/result.json

```diff
@@ -1,10 +1,12 @@
 {
     "context": {
         "data_points": {
+            "images": null,
             "text": [
                 "Benefit_Options-2.pdf: There is a whistleblower policy."
             ]
         },
+        "followup_questions": null,
```

Comment:
I assume the frontend is okay with these being null? Does it match models.ts?

Reply:
This one matches:
followup_questions: string[] | null;

The data_points model isnt correct in models.ts already anyway, not sure why typescript is okay with.